src/channel/pusher-presence-channel.ts

```diff
@@ -5,16 +5,15 @@ import { PresenceChannel } from './presence-channel';
  * This class represents a Pusher presence channel.
  */
 export class PusherPresenceChannel extends PusherChannel implements PresenceChannel {
-
     /**
      * Register a callback to be called anytime the member list changes.
      *
      * @param  {Function} callback
      * @return {object} this
      */
     here(callback): PusherPresenceChannel {
-        this.bind('pusher:subscription_succeeded', (data) => {
-            callback(Object.keys(data.members).map(k => data.members[k]), this.channel);
+        this.on('pusher:subscription_succeeded', (data) => {
+            callback(Object.keys(data.members).map(k => data.members[k]), this.name);
```

Comment:
I don't know if there is any value in passing this.name here? We already know the name I think. The only reason I was passing the Pusher channel is because it has some extra data on it like how many users are subscribed and all of the current users.